bazel/repository_locations.bzl

```diff
@@ -1056,13 +1056,13 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_name = "Bazel rust rules",
         project_desc = "Bazel rust rules (used by Wasm)",
         project_url = "https://github.com/bazelbuild/rules_rust",
-        version = "82b650d5d0709ae4c0ee8584f4ed92112ba11d67",
-        sha256 = "d087851b76204935f7f23c172eb0d136c09720b8484d8151019523652ce77004",
+        version = "635da93206c2761d9db58ed82e194d2af053faf6",
+        sha256 = "aeaa5b98891c646c5292d36f1ce84682f353395aacddafd6c3d6e0db3026eb85",
```

Comment:
im wondering if we should use v0.1.0 which seems to have been released recently (https://github.com/bazelbuild/rules_rust/releases)

Reply:
+1 for `0.1.0` - https://github.com/bazelbuild/rules_rust/releases/tag/0.1.0

Reply:
Nice I hadn't seen that, thanks, updated

Reply:
version `0.2.0` is now out